aws/request/handlers.go

```diff
@@ -20,22 +20,6 @@ type Handlers struct {
 	AfterRetry       HandlerList
 }
 
-// Copy returns of this handler's lists.
-func (h *Handlers) Copy() Handlers {
```

Comment:
I think it really should be implemented somehow differently. Let say if I initialize a client for dynamo db, everything should be initialized only and only once. I can't see there is anything needs to be copied in that already prepared context.